tests/test_dockerfile.py

```diff
@@ -48,10 +46,10 @@ def tearDown(self):
 
     def test_set_cmd_user(self):
         """
-        Ensure that setting a user in the YAML generates a corresponding
+        Ensure that setting 'user' in 'run' section in the YAML generates a corresponding
         USER instruction in the Dockerfile, immediately before the CMD.
         """
-        self.descriptor['user'] = 1347
+        self.descriptor['run']['user'] = 1347
```

Comment:
Oh interesting we default to a non-root UID now. I have no problem with that. This means that `{% if 'user' in run %}` in the jinja will always be true, but it doesn't hurt to have it for robustness.

Reply:
Turns out I was wrong about this.